scripts/db/litestream.yml

```diff
@@ -0,0 +1,7 @@
+access-key-id: ${AWS_ACCESS_KEY_ID}
+secret-access-key: ${AWS_SECRET_ACCESS_KEY}
+
+dbs:
+  - path: /data
+    replicas:
+      - url: s3://${BUCKET}/backup
```

Comment:
This differs from the container definition (`backup` vs  `backups`). Does that matter at all?

Reply:
see above, file has been removed